### PR TITLE
[docs] Add instruction to start docker on Ubuntu

### PR DESCRIPTION
**Pull request by jengeb**
I just tried the installation but docker didn't start automatically. So I had to execute the newly added command in order to get the hello-world verification running.

### CODE REVIEW

**Comment by phemmer** [comment]
Debian (and ubuntu) services are supposed to auto-start on install. If they do not, the issue is the install script.

See #14576 #14883 #12429 #12427

**Comment by andrenarchy** [comment]
Obviously, the auto-start doesn't work for a lot of people (including me) – tested today. Perhaps the new packages mentioned in #14576 don't solve it?

The extra line in the documentation doesn't do any harm if docker is already running. It would just print `start: job is already running: docker`. But if you're new to docker, the extra line can save you time and frustration.

Therefore, I think the PR should be merged until the auto-start issue is actually resolved.

**Comment by jengeb** [comment]
@andrenarchy +1

**Comment by thaJeztah** [comment]
ping @jfrazelle @moxiegirl I'm a bit unsure here; this works out of the box for most, but we do get this reported now and then; any ideas why this doesn't work for some?

**Comment by jfrazelle** [comment]
this is no longer the case w the new debs and if it is happening it is a bug

**Comment by jfrazelle** [comment]
closing, but let me know if you are having to start docker manually with the new debs

**Comment by andrenarchy** [comment]
I just made a clean install on a machine with Ubuntu 14.04 according to the [docs](https://docs.docker.com/docker/installation/ubuntulinux/#installation). The issue remains:

```
$ wget -qO- https://get.docker.com/ | sh
[...]
$ sudo docker run hello-world
Cannot connect to the Docker daemon. Is 'docker -d' running on this host?
$ sudo service docker start
docker start/running, process 30126
$ sudo docker run hello-world
[...]
Hello from Docker.
This message shows that your installation appears to be working correctly.
```
I suggest that this PR is merged until the issue is actually fixed.

@jfrazelle is there an issue/PR for the deb packages? And have you explicitly tested Ubuntu 14.04?

**Comment by jfrazelle** [comment]
you can open a new issue i do not think there is one currently and i will
try to reproduce

**Comment by andrenarchy** [comment]
@jfrazelle it can be reproduced with a vanilla Ubuntu 14.04.3 (64 bit), see #15620.

**Comment by andrenarchy** [comment]
Is there any doubt that this PR is useful? I suggest to merge it until #15620 is fixed – hopefully we all agree that we don't want to waste the valuable time of docker users. :smiley:

**Comment by nschloe** [comment]
After two hours of struggle, I just found the [corresponding bug](https://github.com/docker/docker/issues/15620) and this PR. Why isn't this merged yet?

**Comment by jfrazelle** [comment]
Are you using the new apt-repo at apt.dockerproject.org?

**Comment by jfrazelle** [comment]
Also we are humans, we are far from perfect, a fix will be found and merged
but we need to make sure of a few things first. We get a high volume of
issues and PRs so being patient helps...

**Comment by jengeb** [comment]
Detailed investigation has been done by @andrenarchy in #15620. And adding the following line

``` $ sudo service docker start ```

to docs/installation/ubuntulinux.md (as suggested by others before) is no modification of docker code and would help many users.

**Comment by nschloe** [comment]
> a fix will be found and merged

Until this happens, I'm sure this will cost many more people a fair share of their time unless the heads-up in this PR is merged. See [here](https://github.com/docker/docker/issues/15620) for a bunch of people who've already had that problem.

**Comment by andrenarchy** [comment]
I suggest to move the discussion of the deb issue to #15620. This PR is for fixing the docs until #15620 is resolved.

@jfrazelle Nobody questioned that the docker team is human. ;) I'm sure you'll find and fix issue #15620 in the future. All we ask for is that *this* PR is merged *now*. Can you do this?

**Comment by frehner** [comment]
@jfrazelle Really, is there a reason why this documentation PR is not being merged?  Look at how many people are running into this issue and how much time could be saved by simply merging this small change to the docs.  Once the deeper issue is resolved, then you can roll this back, but until then, please help people save hours of their time by simply adding this one instruction. :)

**Comment by jfrazelle** [comment]
ill leave it to the docs team to decide fine by me either way

**Comment by thaJeztah** [comment]
I'm good with reopening this PR, after all, it won't do anything if the daemon is already started.

(Still interested to know *what* is causing this to fail for *some* but not all users).

File: docs/installation/ubuntulinux.md [hunk 1]
```diff
@@ -127,7 +127,11 @@ install Docker using the following:
     >
     >       $ wget -qO- https://get.docker.com/gpg | sudo apt-key add -
 
-4. Verify `docker` is installed correctly.
+4. Start `docker`.
```
Comment (thaJeztah) [inline docs/installation/ubuntulinux.md]: @jengeb Can you change this to "Start the Docker daemon"?

**Comment by thaJeztah** [comment]
One comment, but LGTM once that's addressed, thanks!

**Comment by jengeb** [comment]
@thaJeztah I added the new comment.

**Comment by thaJeztah** [comment]
Thanks! LGTM

ping @SvenDowideit for review

**Comment by moxiegirl** [comment]
LGTM

**Comment by andrenarchy** [comment]
This change is not yet live on https://docs.docker.com/installation/ubuntulinux/#installation. Can you deploy the documentation?

**Comment by thaJeztah** [comment]
@andrenarchy last thing I heard is that the next docs publish will be done along with the 1.9 release, so unfortunately, not yet.

We're also in the process of rewriting the docs to move away from the curl/sh installation (see  https://github.com/docker/docker/pull/16662)

**Comment by davetcoleman** [comment]
This is silly, wasting lots of people's time with a bug like this that just needs easy documentation

**Comment by phemmer** [comment]
@davetcoleman This PR has been merged.

**Comment by davetcoleman** [comment]
Yes, but it is not live on the website so its as if it wasn't merged

**Comment by andrenarchy** [comment]
This PR makes me sad. So much wasted time for nothing. Why can't the docs be updated?

**Pull request merged**